packages/client/src/store/modules/grants.js

```diff
@@ -164,10 +178,10 @@ export default {
     async setEligibilityCodeEnabled(context, { code, enabled }) {
       await fetchApi.put(`/api/organizations/:organizationId/eligibility-codes/${code}/enable/${enabled}`);
     },
-    fetchSavedSearches({ commit }) {
+    async fetchSavedSearches({ commit }) {
       // TODO: Add pagination URL parameters.
-      fetchApi.get('/api/organizations/:organizationId/grants-saved-search')
-        .then((data) => commit('SET_SAVED_SEARCHES', data));
+      const data = await fetchApi.get('/api/organizations/:organizationId/grants-saved-search');
```

Comment:
So we can make sure new saved search exists when loading other elements  from it